app/Helpers/database/user-activity.php

```diff
@@ -388,6 +285,7 @@ function getLatestRichPresenceUpdates(): array
     return $playersFound;
 }
 
+// TODO dead code?
```

Comment:
open TODO

Reply:
I think it used to be used for recently added achievements on the main page. Now we have recently added sets.